view.go

```diff
@@ -610,10 +611,11 @@ func (v *View) draw() error {
 				x:       x,
 				y:       y,
 			})
-			if err := v.setRune(x, y, char.chr, fgColor, bgColor); err != nil {
+			chr, err := v.setRune(x, y, char.chr, fgColor, bgColor)
+			if err != nil {
 				return err
 			}
-			x += runewidth.RuneWidth(char.chr)
+			x += runewidth.RuneWidth(chr)
```

Comment:
This should be fixed differently to be honest. It can use the original code and just check for the NULL character how it was doing before.

But the cell buffer needs to be fixed across the project. I've already mentioned this in the other PR which is addressing UTF8 wide characters.    
I let you know when the PR so you could check if it's ok like that.